shared/spikestatsOverTime.m

```diff
@@ -0,0 +1,448 @@
+function [stats, legend_out] = spikestatsOverTime(cfg, spikedata)
```

Comment:
similarly as previous function, too much and too specific for /shared; please move to project directory.